app/src/main/java/de/tum/in/tumcampusapp/component/other/generic/activity/BaseActivity.kt

```diff
@@ -41,4 +48,30 @@ abstract class BaseActivity(
 
         return super.onOptionsItemSelected(item)
     }
+
+    // load language from user's preferences
+    private fun initLanguage(context: Context) {
+        val sharedPreferences = PreferenceManager.getDefaultSharedPreferences(context)
+        var lang = sharedPreferences.getString("language_preference", null)
```

Comment:
I think we can default this to "en" and just ignore the whole if block below. English will always be in there.

Reply:
Ah, makes perfect sense! Of course I did not think about that "initial" phase.